src/dataCollection.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python
+"""
+@file dataCollection.py .
+
+@author Ayush Saini
```

Comment:
Please do `class` based version no code will be accepted without class and it's doc's as in robro cloud

Reply:
I guess no need for class, it's a simple script.

Reply:
see it's needed we implement most of all the `ROS` code in class also we need to standardize the coding style as well as docstrings so for that thing we need all things inside a class. So we can make a more abstract layer on it